scripts/docs_link_checker.py

```diff
@@ -0,0 +1,306 @@
+import glob
+import logging
+import os
```

Comment:
Could you write a small file-level docstring explaining the purpose of this script and/or how to use it?

Reply:
done!